common/docker/client_container.go

```diff
@@ -258,10 +271,10 @@ func (c *Client) logContainerOutput(ctx context.Context, containerID, styleNumbe
 		// Print the cleaned log message
 		if streamType == 1 { // Stdout
 			// TODO: what content should be printed for stdout?
-			fmt.Printf("[%s] %s", style.ForegroundPrint(containerID, styleNumber), cleanLog)
+			fmt.Printf("[%s] %s", style.ForegroundPrint(containerID, colors[styleNumber]), cleanLog)
 		} else if streamType == 2 { //nolint:gomnd // Stderr
 			// TODO: what content should be printed for stderr?
-			fmt.Printf("[%s] %s", style.ForegroundPrint(containerID, styleNumber), cleanLog)
+			fmt.Printf("[%s] %s", style.ForegroundPrint(containerID, colors[styleNumber]), cleanLog)
```

Comment:
Since we're removing red from container name colors, we can use it here. Red is a strong color though, and doesn't contrast well with a black background. I suggest keeping the log message in black/white and change the `[${containerName}]` color instead.

Another idea is to change the bg color, this will sort of "highlight" the err. might need to experiment

Reply:
Actually the log message is still using default color here, only the name of the container using different color each.